### ISSUE TITLE
For multi-select drop down when all options are checked focus for the options are not clearly visible

### ISSUE THREAD

**Issue by SainathBommisetty** [microsoft/fluentui#12121]
### Environment Information

Browser and OS versions: Chrome,Edge

### Describe the issue:

In multi-select Dropdown when all the options are checked focus for the options are not clearly visible.

### Steps:

1.Select multi-select Dropdown and navigate using keyboard arrow keys
2. check all the options appear in the Dropdown and navigate using keyboard arrow keys
3. while navigating the focus should be clearly visible

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/HarikaAlapati/pen/QWbvxaX

### Actual behavior:

Focus indicator is not clearly visible on the options of the "Multi-select Dropdown" due to which keyboard user will face difficulty in finding the location of the keyboard focus indicator

### Expected behavior:

Focus should be clearly visible after selecting the check boxes.

### Unchecked focus on carrot:

![image](https://user-images.githubusercontent.com/17314739/75548457-53076380-5a53-11ea-8607-58e4c36c1648.png)

### Checked focus on carrot: 

![image](https://user-images.githubusercontent.com/17314739/75548560-9792ff00-5a53-11ea-83c8-9ed61dd16be2.png)

**Comment by aneeshack4** [comment]
Thanks for reaching out to us @SainathBommisetty! I repro this and I see that we use a different way to show focus in the multi select combobox that works better - speaking with the component owner to see if we can do something similar on Dropdown or fix the existing border disappearing during keyboarding some other way.

**Comment by aneeshack4** [comment]
@SainathBommisetty I spoke with component owner @joschect. He currently has a PR out #11900 that should fix some of this but said that as of now there's isn't a good complete fix. We'll add this to the backlog and work on it further - it needs design and so will take some time. Thank you for your patience.
